src/filter/filters.ts

```diff
@@ -231,8 +239,11 @@ export function apply(
   for (const key in filters) {
     if (!Object.hasOwnProperty.call(filters, key)) continue
     if (key in preprocessors) {
-      if (!context) return false
```

Comment:
If we use these wide-band try-catch blocks it should handle this case. We could also leave this in just to avoid the error, sort of neutral on this.